vulture_allowlist.py

```diff
@@ -299,6 +303,7 @@
 IAgentEventRepository.get_events_by_type
 IAgentEventRepository.get_events_by_tag
 IAgentEventRepository.get_events_by_source
+MongoAgentEventRepository
```

Comment:
Vulture complains about this? It's used in `monkey_island/cc/services/initialize.py`.